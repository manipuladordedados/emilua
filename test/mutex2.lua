-- Emilua's mutex is a subset of PTHREAD_MUTEX_ERRORCHECK
-- Therefore, EPERM should be raised here

local mutex = require('mutex')

m = mutex.new()
m:unlock()
