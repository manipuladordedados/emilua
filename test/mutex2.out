Main fiber from VM 0x1 panicked: 'Operation not permitted'
stack traceback:
	input.lua:7: in main chunk
